### ISSUE TITLE
[Split Button] Ensure the button used for actions includes ARIA roles, states, and properties are valid (CAL-AMP31)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13535]
**monday.com sync:** #10821956033

### Violation: 
 Ensure ARIA roles, states, and properties are valid 

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html) 

### Recommend solution
`split-button` should provide more context on its child `button` which includes secondary "actions". The context should be provided to assistive technologies using the [`aria-haspopup="listbox"`](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/listbox/listbox-collapsible.html), and some cleanup of `aria-`named attributes may also need be included.

### Severity: 
 High 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP31 Ensure ARIA roles, states, and properties are valid 
- Module: M4 Form Elements_Employee Dashboard 
- Finding ID: [CAL-AMP31](https://esri.hub.essentia11y.com/short-link/8XU-CA_d0uhPmGae) 

**Issue**
There is a "v" (expand control present besides "Form Actions") with unnecessary aria-live and aria-busy attributes.

**User Impact**
Assistive technologies may be unable to parse this content, so users may not be able to perceive the feature(s) or control(s) correctly, or may not be able to operate them.

**Code Reference** 
 ```
<button aria-live="polite" aria-busy="false" aria-label="Save or Submit the form" class=" button-padding--shrunk icon-end-empty " href="false" rel="false" target="false" type="button">(...)</button>

```

**Steps to Reproduce**
1. Open Chrome DevTools.
2. Locate the element identified in the code reference.
3. Notice that the element's ARIA attributes and/or nesting do not conform to the ARIA specification: https://www.w3.org/TR/wai-aria/ and/or the ARIA in HTML specification: https://www.w3.org/TR/html-aria/
 
**Recommendation**
Ensure ARIA roles, states, and properties are valid. 

In this case, the developer should remove unnecessary aria attributes and provide aria-haspopup to the control.

**Compliant Code Example**
```
<img src="oldbarn.png" alt="Old barn near Jenny Lake" aria-details="imgDetails">
...
<p id="imgDetails">
An old barn with a tin roof and weathered siding stands bathed in morning sunlight near Jenny Lake with the Wyoming Rocky Mountains in the background.
</p>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP31.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/8XU-CA_d0uhPmGae) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
